cron.ts

```diff
@@ -1,77 +1,103 @@
 import { IsNull, Not } from "typeorm";
 import config from "./config";
-import { DiscordMemberRepository, DiscordServerRepository } from "./db";
+import { DiscordMemberRepository, DiscordServerConfigRepository } from "./db";
 import { DiscordMember } from "./db/entity/DiscordMember";
-import { DiscordServer } from "./db/entity/DiscordServer";
+import { DiscordServerConfig } from "./db/entity/DiscordServerConfig";
 import { restDiscordClient } from "./discord/client";
 import { addRole, removeRole } from "./discord/role";
 import modules from "./starkyModules";
 import { StarkyModule } from "./starkyModules/types";
 
-const refreshDiscordServers = async () => {
-  const discordServers = await DiscordServerRepository.find();
-  for (let discordServer of discordServers) {
-    await refreshDiscordServer(discordServer);
+const refreshDiscordServerConfigs = async () => {
+  const DiscordServerConfigs = await DiscordServerConfigRepository.find();
+  for (let DiscordServerConfig of DiscordServerConfigs) {
+    await refreshDiscordServerConfig(DiscordServerConfig);
   }
 };
 
-export const refreshDiscordServer = async (discordServer: DiscordServer) => {
-  console.log(`[Cron] Refreshing discord server ${discordServer.id}`);
+export const memberStarknetAddress = async (discordMember: DiscordMember) => {
```

Comment:
- naming is not clear: this method actually updates the `discordMember`
- also I don't understand why it's needed? Let's discuss but I think the `discordMember` should be linked to a single entity `discordServer` and a `discordServer` should have multiple configs?